### ISSUE TITLE
469d running really choppy compared to 469c

### ISSUE THREAD

**Issue by Captrico** [OldUnreal/UnrealTournamentPatches#1481]
I was using 469c on my system. Its not high end but it ran really well.   Its a Acer Veriton 4630g  i5, Intel HD 4600 on board.  The issue im having is a jerky sort of strobing effect when in a fire fight, and most noticeably when using the redeemer. Its as if the game is skipping a frame between frames, if that makes sense. Not very fluid at all.  The frame rate is around 50 ish.  The only issue i had with 469c was the smoke from the rocket fire would lag the game a bit, but i think that was my pc struggling with the process.   This issue now is really a bummer, because i cant seem to load the old 469c back over the 469d patch, in order to see if thats the problem.   My ping also seems slower. Before i was at around 70 and now its around 130.    Not sure whats up but i miss my Deck 16 deathmatch. :(

**Comment by SeriousBuggie** [comment]
You can install v469c on top of 469d for return back and check if problem still here.
You specify which video and audio renders you use in game.

**Comment by Captrico** [comment]
I think i figured it out....maybe.  UT99 on you tube said to load the hotfix in again, and that seemed to smooth things up alot more.  I think i may have forgotten that step.  Either way i may be back on track! :)

**Comment by Captrico** [comment]
Is there a way to run SAFE MODE from the system files?  I changed a setting on the drivers and the screen is now mostly white and is running in the background.  When i boot the game i assumed it would give me an option to change drivers, but it goes straight into the game, with the white screen.  I cant see the menu tabs to change the thing back that messed up the display. :D  It was running OK in XOpenGL which was surprising, because i used OPEN GL before and had issues with X.    Now i just need to get in SAFE MODE so i can reset the darn thing.

**Comment by SeriousBuggie** [comment]
You can just create Running.ini in System folder and run game. This trigger safe mode.
Or you can run UnrealTournament.exe with switch ` -changevideo`
Also you can use switch `-safe` for call safe mode directly.

**Comment by Captrico** [comment]
I found that logging off while game is running also prompts safe mode when re booting.    I cant seem to get 469c to stick when i load it back over 469d.  It still says 469d on the top of the ut99 menu screen for some reason.   I did load the hotfix and it worked but had some flickers now and again on the menu screen.   I can get XOPENGL to run fine, but it crashes whenever i load the game again later.  Says there is a GL error 'critical'  and i have to go back in safe mode, defaults settings, but i have to reset everything all over again.  My video is Intel HD 4600 , onboard, so maybe it has issues with things.   469c worked fine for the most part.  It all went downhill after i loaded 469d.  Now if i try switching between the 2 patches it stays the same.  I would re install the whole game but i have already loaded files into my downloaded copy with 469d and i dont see the UT99 Community HD 469c zip on Archive.org anymore.   Just the UT99 469d.

**Comment by SeriousBuggie** [comment]
Install patch from here, sip version, with replace all files, must switch it 469c.
Intel is very well known for bad drivers. I suggest you use d3d9/d3d11 renderers.
You can find two recent d3d11 drivers here:

https://github.com/dpjudas/UT99VulkanDrv

https://github.com/metallicafan212/ICBINDx11Drv

**Comment by Captrico** [comment]
I tried the d3d9 which shows on my device compatible drivers.  It seems to run OK.  Maybe not as smooth as XOpenGL , but X has problems when i try to play the game from boot up.  These 2 drivers you showed me......do i install into UT99 or onto my Acer PC?

**Comment by Captrico** [comment]
![Screenshot (58)](https://github.com/OldUnreal/UnrealTournamentPatches/assets/153848409/809aa03c-ddd7-4ad1-808f-8c9bdf54582e)

heres what im using on my pc

**Comment by SeriousBuggie** [comment]
Into your UT99. it is not driver. It is renderer. Thing, which game for render picture.

**Comment by Captrico** [comment]
so, copy and paste this information?  
![Screenshot (61)](https://github.com/OldUnreal/UnrealTournamentPatches/assets/153848409/3133b67e-aa21-4764-903f-11a74642da8f)
![Screenshot (60)](https://github.com/OldUnreal/UnrealTournamentPatches/assets/153848409/45af4376-e7cd-411d-a5be-92cb5b680461)

**Comment by SeriousBuggie** [comment]
Basically you need goes to release page, download zip with appropriate release (usually latest) and install (unpack) to game folder. After that in game choose change video renderer. Select new installed renderer and use it.
Edit config is optional. Renderer create section in config if not found it. With some default values.

**Comment by Captrico** [comment]
hi,

well i downloaded the 469d community game, the whole game. Started fresh.    I used direct 3d 9 and it seems to run much better now.  I changed some settings in advanced display options.   Very smooth and no jittering or skips. :D     I think maybe i had too many programs on each other in the other 469c game, i dont know.   This seems to run great now.    I can play again!! Thanks for your help.
